### PR TITLE
Add Profile table migration

### PR DESCRIPTION
**Pull request by ramos-ph**
This PR will add a profile table migration for Knex so we can start the development of some basic features.

I also fixed the config in `knexfile.js`, added `dotenv` in the project, and fixed the connection string on `.env.example`.

### CODE REVIEW

File: src/database/migrations/20220813005900_create_profiles_table.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = (knex) => knex.schema.createTable('profiles', (table) => {
+  table.increments('id');
+
+  table.string('name').notNullable();
+  table.string('username').notNullable();
+  table.string('password').notNullable();
+  table.string('avatar_url').nullable();
```
Comment (noccss) [inline src/database/migrations/20220813005900_create_profiles_table.js:11]: What's avatar_url?

Comment (ramos-ph) [reply src/database/migrations/20220813005900_create_profiles_table.js:11]: It's a reference to the profile avatar on the server. We don't usually save images on the database, but instead we store them in a `/public` directory on the server and point to it on the front-end.

When we develop the profile creation, we'll upload the avatar as a file and store it on the server, but only the URL for reference will be saved on the database.

Comment (JuanTrindade) [reply src/database/migrations/20220813005900_create_profiles_table.js:11]: Lgtm

Comment (ramos-ph) [reply src/database/migrations/20220813005900_create_profiles_table.js:11]: In sum, we'll do the following:
```html
<img src="<url of the image on the server>" />
```

**Review by JuanTrindade — COMMENTED** [review]
The dotenv does what?

File: src/database/migrations/20220813005900_create_profiles_table.js (hunk 1 above)
Comment (noccss) [reply src/database/migrations/20220813005900_create_profiles_table.js:11]: Nice, now i understand.
Thank you!

**Review by noccss — APPROVED** [review]

**Comment by ramos-ph** [comment]
> The dotenv does what?

It reads the `.env` file and loads them on the application, more specifically, in `process.env`.
Without it, the values we configured, like `PG_CONNECTION_STRING`, will be unavailable in our app.

**Comment by JuanTrindade** [comment]
> > The dotenv does what?
> 
> It reads the `.env` file and loads them on the application, more specifically, in `process.env`. Without it, the values we configured, like `PG_CONNECTION_STRING`, will be unavailable in our app.

**Pull request merged**